src/main/java/mate/academy/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
+    private long id;
```

Comment:
Let's keep all fields as objects in here.
```suggestion
    private Long id;
```